arcgis-ios-sdk-samples/Layers/Group layers/GroupLayersSectionView.xib

```diff
@@ -24,7 +22,7 @@
                 <switch opaque="NO" contentMode="scaleToFill" horizontalHuggingPriority="750" verticalHuggingPriority="750" contentHorizontalAlignment="center" contentVerticalAlignment="center" on="YES" translatesAutoresizingMaskIntoConstraints="NO" id="9ku-4w-xzb">
                     <rect key="frame" x="310" y="6.5" width="51" height="31"/>
                     <connections>
-                        <action selector="toggleSwitch:" destination="-2" eventType="valueChanged" id="ndi-hb-abW"/>
```

Comment:
The ol' destination -2 bug. Gets ya every time. 🙄

Reply:
Ohh I'm also quite curious - I tried to search for it but couldn't find the right keyword. 😲  Is it a well-known bug?

Reply:
I've never heard of it before! 😂